spec/controllers/about_controller_spec.rb

```diff
@@ -1,4 +1,6 @@
 RSpec.describe AboutController do
+  render_views
```

Comment:
(Without this, it seems to show "relevant lines: 1" for all views in question, rather than actually testing them)